src/pages/home/report/ReportActionsView.js

```diff
@@ -145,6 +165,33 @@ class ReportActionsView extends React.Component {
         this.setState({refetchNeeded});
     }
 
+    /**
+     * Checks if the unreadActionIndicator should be shown.
+     * If it does, starts a timeout for the fading out animation and creates
+     * a flag to not show it again if the report is still open
+     */
+    setUpUnreadActionIndicator() {
+        if (!this.props.isActiveReport || !this.shouldShowNewActionIndicator) {
```

Comment:
This is a clever workaround to not being able to use `componentDidMount()`. Hopefully we can move this method there in the near future :)